File: peel/src/main/java/net/tjalp/peach/peel/config/RedisDetails.java
package net.tjalp.peach.peel.config;

/**
 * The details for the redis connection
 */
public class RedisDetails implements Configurable {

    /** The target address for redis */
    public String address = "127.0.0.1";

    /** The target port for redis */
    public int port = 6379;

    /** The password for redis */
    public String password = "";
}

File: peel/src/main/kotlin/net/tjalp/peach/peel/config/RedisDetails.kt
package net.tjalp.peach.peel.config

/**
 * The details for the redis connection
 */
class RedisDetails {

    /** The target address for redis */
    var server = "host.docker.internal"

    /** The target port for redis */
    var port = 6379

    /** The password for redis */
    var password = ""
}